front/app/components/admin/InviteUsersWithSeatsModal/index.tsx

```diff
@@ -31,6 +39,25 @@ const InviteUsersWithSeatsModal = ({
   const { formatMessage } = useIntl();
   const [hasAcknowledged, setHasAcknowledged] = useState(false);
   const [showWarning, setShowWarning] = useState(false);
+  const [showSuccess, setShowSuccess] = useState(false);
+  const { data: appConfiguration } = useAppConfiguration();
+  const { data: seats } = useSeats();
+
+  if (!appConfiguration || !seats) return null;
+
+  const maximumSeatNumber = {
+    admin:
+      appConfiguration?.data.attributes.settings.core.maximum_admins_number,
+    collaborator:
+      appConfiguration?.data.attributes.settings.core.maximum_moderators_number,
+  }[seatType];
+  const currentSeatNumber = {
+    admin: seats.data.attributes.admins_number,
+    collaborator: seats.data.attributes.project_moderators_number,
```

Comment:
Not blocking, but very easy to introduce typos without types. This will pass without complaints:

`admim: seats.data.attributes.admins_number`